x-pack/plugins/security_solution/public/hosts/components/authentications_table/translations.ts

```diff
@@ -7,7 +7,7 @@
 import { i18n } from '@kbn/i18n';
 
 export const AUTHENTICATIONS = i18n.translate(
-  'xpack.securitySolution.authenticationsTable.authenticationFailures',
+  'xpack.securitySolution.authenticationsTable.authenticationsFailures',
```

Comment:
Could we change this to `xpack.securitySolution.authenticationsTable.authentications` instead